corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -82,7 +82,7 @@ def filter_cases(self, xpath, additional_filters=None, auth=None, extra_instance
         response = post_data(
             json.dumps(data),
             url=url,
-            content_type="text/json", auth=auth
```

Comment:
ah i see now, and this works fine with the old touchforms? it's possible that the old touchforms needs this